roles/ceph-iscsi-gw/tasks/common.yml

```diff
@@ -2,7 +2,16 @@
 - name: make sure gateway_ip_list is configured
```

Comment:
I think these 2 tasks should be moved into ceph-validate role

Reply:
This should be handled now. I moved the checks to ceph-validate like the iscsi connections is.